packages/client/src/links/httpSubscriptionLink.ts

```diff
@@ -47,6 +50,17 @@ type HTTPSubscriptionLinkOptions<TRoot extends AnyClientTypes> = {
 } & TransformerOptions<TRoot> &
   UrlOptionsWithConnectionParams;
 
+/**
+ * tRPC error codes that are considered retryable
+ * With out of the box SSE, the client will reconnect when these errors are encountered
+ */
+const codes5xx: TRPC_ERROR_CODE_NUMBER[] = [
+  TRPC_ERROR_CODES_BY_KEY.BAD_GATEWAY,
+  TRPC_ERROR_CODES_BY_KEY.SERVICE_UNAVAILABLE,
+  TRPC_ERROR_CODES_BY_KEY.GATEWAY_TIMEOUT,
+  TRPC_ERROR_CODES_BY_KEY.INTERNAL_SERVER_ERROR,
+];
```

Comment:
cc @Nick-Lucas it will now `throw` whenever it encounters a serialized error that isn't one of these, that matches what Chrome does whenever it encounters a HTTP-error

For your usecase, let's do a [`retryLink`](https://github.com/trpc/trpc/blob/next/packages/client/src/links/internals/retryLink.ts) instead

Reply:
retryLink does sounds worth exploring!